### PR TITLE
you forgot -moz-fit-content again wl

### PR DESCRIPTION
**Pull request by RedGuy12**
also idk why `-moz-crisp-edges` wasn't caught last time
<details>
<summary>prefixes we need to use</summary>

Browsers:
  Chrome for Android: 98
  Firefox for Android: 96
  Chrome: 98, 97, 96, 95, 94, 93, 92, 91, 90, 89, 88, 87, 86, 85, 84, 83, 81, 80
  Edge: 98, 97, 96, 95, 94, 93, 92, 91, 90, 89, 88, 87, 86, 85, 84, 83, 81, 80
  Firefox: 97, 96, 95, 94, 93, 92, 91, 90, 89, 88, 87, 86
  Opera: 83, 82, 81, 80

These browsers account for 71.28% of all users globally

Selectors:
  ::file-selector-button: webkit
  :autofill: webkit

Properties:
  appearance: webkit
  background-clip: webkit
  box-decoration-break: webkit
  color-adjust: webkit
  mask-border-outset: webkit
  mask-border-repeat: webkit
  mask-border-slice: webkit
  mask-border-source: webkit
  mask-border-width: webkit
  mask-border: webkit
  mask-clip: webkit
  mask-composite: webkit
  mask-image: webkit
  mask-origin: webkit
  mask-position: webkit
  mask-repeat: webkit
  mask-size: webkit
  mask: webkit
  tab-size: moz
  text-emphasis-color: webkit
  text-emphasis-position: webkit
  text-emphasis-style: webkit
  text-emphasis: webkit
  text-size-adjust: moz

Values:
  cross-fade: webkit
  element: moz
  fill-available: webkit
  fill: webkit
  fit-content: moz
  image-set: webkit
  pixelated: moz
  stretch: webkit, moz

</details>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
-# Files created during packaging
-.manifest.js.bak
-.dist
+# Files created during packaging
+.manifest.js.bak
+.dist
+
+.browserslistrc
```
Comment (RedGuy12) [inline .gitignore:5]: wl said he doesn't want it committed so i want it ignored lol

File: addons/gamepad/style.css [hunk 2]
```diff
@@ -40,6 +40,7 @@
   will-change: transform;
   image-rendering: optimizeSpeed;
   image-rendering: crisp-edges;
+  image-rendering: -moz-crisp-edges;
```
Comment (WorldLanguages) [inline addons/gamepad/style.css:43]: According to MDN, `crisp-edges` is enough on Firefox 65+

Comment (WorldLanguages) [reply addons/gamepad/style.css:43]: Also, I think garbo meant `text-rendering: optimizeSpeed`? I don't think `optimizeSpeed` is valid here.

**Comment by WorldLanguages** [comment]
Git/GitHub bug, apparently I can't prevent this PR from "modifying" the `.gitignore` file even tho its contents haven't changed at all.

**Comment by RedGuy12** [comment]
> Git/GitHub bug, apparently I can't prevent this PR from "modifying" the `.gitignore` file even tho its contents haven't changed at all.

i think i changed the line ending maybe

**Comment by WorldLanguages** [comment]
@RedGuy12 The file is identical. I tried git checkout and it wouldn't let me commit because there were no changes.

**Comment by WorldLanguages** [comment]
Replaced by #4373

**Pull request closed without merging**